src/jobs/blue_green.yml

```diff
@@ -0,0 +1,94 @@
+description: >
+  Execute a blue/green deploy  in a single job. Expects either build_steps
+  or workspace_path for assets to deploy.
+docker:
+  - image: circleci/node:10
```

Comment:
hardcoded image on the old namespace. Recommend changing with image parameter and defaulting to cimg